lib/medium-admin.php

```diff
@@ -136,6 +136,12 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters("medium_allowed_post_types", array("post"));
+
+    if (!in_array($post->post_type, $allowed_post_types)) {
```

Comment:
can you remove braces and do the return on a single line to match the rest of the file, and go ahead and remove the space above it to.